docs/Coding-Guidelines/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant
+motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
+expected)
+- [ ] This change requires a documentation update
```

Comment:
SUGGESTION:

Specify the nature of the change (select all that apply):

- [ ] Breaking Change (source, ABI and/or behavior incompatibility)
- [ ] Bug Fix (corrective change fixing issue(s))
- [ ] Enhancement (improvement or other change to APIs and/or behavior)
- [ ] Documentation
- [ ] Test-only (no product impact)